src/main/java/hudson/plugins/ec2/EC2AbstractSlave.java

```diff
@@ -459,6 +466,11 @@ public Node reconfigure(final StaplerRequest req, JSONObject form) throws FormEx
         return null;
     }
 
+    @Override
+    public boolean isAcceptingTasks() {
```

Comment:
The addition of this method has resulted in a critical bug (as noted by @nigelarmstrong-bc) - this method does not exist on serialized versions of this class created by previous plugins, and you get this exception in Jenkins on any access:

```
Caught exception evaluating: c.iconClassName in /jenkins/ajaxExecutors. Reason: java.lang.reflect.InvocationTargetException
java.lang.NullPointerException
	at hudson.plugins.ec2.EC2AbstractSlave.isAcceptingTasks(EC2AbstractSlave.java:471)
	at hudson.model.Computer.isAcceptingTasks(Computer.java:1612)
	at hudson.slaves.SlaveComputer.isAcceptingTasks(SlaveComputer.java:177)
	at hudson.model.Computer.getIconClassName(Computer.java:753)
Caused: java.lang.reflect.InvocationTargetException
```

Reply:
cc @thoulen ^^